app/javascript/react/components/molecules/StationValueLabel/StationValueLabel.style.tsx

```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components";
+
+const GradientBox = styled.div`
+  background: linear-gradient(
+      241deg,
+      rgba(0, 181, 239, 0.4) -2.4%,
+      rgba(0, 181, 239, 0) 94.94%
+    ),
+    #81dbcb;
+  position: relative;
+  display: flex;
+  color: white;
+  width: 16%;
+  height: 11%;
+  border-radius: 10px;
+`;
+
+const GradientImage = styled.div`
+  position: absolute;
+  flex: 1;
+  display: flex;
+  align-items: left;
+  left: 6%;
+  top: 50%;
+  transform: translateY(-50%);
+  height: 60%;
+  width: 40%;
+`;
+
+const TextContainer = styled.div`
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  align-items: right;
+  text-align: right;
+  padding-right: 6%;
+  padding-top: 10%;
+  padding-bottom: 10%;
+  color: white;
+`;
+
+export { GradientBox, GradientImage, TextContainer };
```

Comment:
I needed to refactor something and this is what I ended up with:

```
import styled from "styled-components";

const GradientBox = styled.div`
  background: linear-gradient(
      241deg,
      rgba(0, 181, 239, 0.4) -2.4%,
      rgba(0, 181, 239, 0) 94.94%
    ),
    #81dbcb;
  display: flex;
  flex-direction: row;
  align-items: flex-end;
  justify-content: space-between;
  padding: 34px 20px;
  width: 290px;
  height: 180px;
  border-radius: 10px;
`;

const GradientImage = styled.img`
  height: 115px;
  width: 115px;
`;

const TextContainer = styled.div`
  flex: 1;
  display: flex;
  flex-direction: column;
  text-align: right;
  color: white;
`;

const Label = styled.span`
  font-size: 16px;
  font-weight: 400;
`;

export { GradientBox, GradientImage, TextContainer, Label };
```
and then:
```
import React from "react";
import BroadCastLogo from "../../../assets/icons/broadCastLogo.svg";
import { LargeH1 } from "../../../assets/styles/Typography";
import * as S from "./StationValueLabel.style";

interface StationValues {
  date: string;
  value: number;
  parameter: string;
}

const StationValueLabel = ({ date, value, parameter }: StationValues) => {
  return (
    <S.GradientBox>
      <S.GradientImage
        src={BroadCastLogo}
        alt="Broadcasting logo"
      ></S.GradientImage>
      <S.TextContainer>
        <S.Label>Avg for {date}</S.Label>
        <LargeH1>{value}</LargeH1>
        <S.Label>{parameter}</S.Label>
      </S.TextContainer>
    </S.GradientBox>
  );
};

export { StationValueLabel };
```
Still this LargeH1 is to be changed as described elsewhere.
It will be hard to describe it, we can have a huddle and talk about the choices. :)

Reply:
Thank you!